Cargo.toml

```diff
@@ -10,6 +18,179 @@ members = [
+pallet-transaction-payment-rpc             = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-basic-authorship                        = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-chain-spec                              = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-cli                                     = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-client-api                              = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-consensus                               = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-consensus-aura                          = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-executor                                = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-keystore                                = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-network                                 = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-network-common                          = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-offchain                                = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-rpc                                     = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-rpc-api                                 = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-service                                 = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-sysinfo                                 = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-telemetry                               = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-tracing                                 = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-transaction-pool                        = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sc-transaction-pool-api                    = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sp-blockchain                              = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sp-timestamp                               = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+sp-consensus                               = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
```

Comment:
How do you know these dependencies should not be `default-features = false`?

And this will break the Subalfred features checker.

Reply:
For the dependencies used in the runtime, add `default-feature = false`, otherwise do not need it.

The compiler will fire if something is wrong.